include/cpr/digest.h

```diff
@@ -7,12 +7,7 @@ namespace cpr {
 
 class Digest : public Authentication {
   public:
-    Digest(std::string&& username, std::string&& password)
-            : Authentication{std::move(username), std::move(password)} {}
-
-    ~Digest() override = default;
-
-    const char* GetAuthString() const noexcept;
+    using Authentication::Authentication;
```

Comment:
I'm not a fan of this notation.

Could you please replace it with the appropriate overrides for constructors.

Reply:
I don't understand "not a fan" -- this is a bog standard C++11 inherited constructor.

Reply:
I put explicit constructors back in.

Reply:
Without the explicit constructors I have to open two files to find out, which constructors are provided.
So this is more work for me as a developer using `CPR`.